support-frontend/app/utils/NormalisedPhoneNumber.scala

```diff
@@ -11,36 +11,27 @@ case class NormalisedTelephoneNumber(countryCode: String, localNumber: String)
 
 object NormalisedTelephoneNumber {
 
-  def asFormattedString(normalisedTelephoneNumber: NormalisedTelephoneNumber): String =
+  private def asFormattedString(normalisedTelephoneNumber: NormalisedTelephoneNumber): String =
```

Comment:
tidy up in this file to make it a single thing to call from the one place it's used in the code.  Now we just have to call `formatFromStringAndCountry` once and we're done.  The last method is used in tests so I kept it mostly as is.